src/messages/components/ConversationAddReaction.vue

```diff
@@ -7,24 +7,28 @@
       anchor="top left"
       self="top left"
       @show="open = true"
-      @hide="open = false"
+      @hide="open = false, search = ''"
     >
+      <QSearch
```

Comment:
Can you move this inside `ConversationAddReactionInner`? Then the search term should be cleared implicitly when the popover gets closed (because of `v-if="open"`).

Reply:
Sure, I can do that.